generator.yaml

```diff
@@ -1,2 +1,33 @@
 ignore:
-  resource_names:
+  field_paths:
+    - "RequestCertificateInput.IdempotencyToken"
+operations:
+  RequestCertificate:
```

Comment:
Do you need to mark the `DomainName` field as `is_required` here?  It's a required field, yet I don't see anything in the `sdkCreate` method enforcing it as a required field.

Reply:
@jljaco DomainName is marked as a required field in the API model (api-2.json file in aws-sdk-go) and therefore the CRD's `Spec.DomainName` field is annotated with this automagically:

```go
// +kubebuilder:validation:Required
```

What that does is generate the OpenAPI Validating Schema that is loaded into the Kubernetes API server and provides before-reconciler validation logic that would catch these types of simple validations.

You can see this OpenAPI Validating Schema if you see the [`config/crd/bases/acm.services.k8s.aws_certificates.yaml`](https://github.com/aws-controllers-k8s/acm-controller/pull/2/files#diff-2e9d9a6aa61fe4f842a6f1471daf9cae8166d3d1f944a94cf364e27fd3be9db5) file and look for:

```json
            required:
            - domainName
```